src/pages/RocketsPage.jsx

```diff
@@ -1,6 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
- [ ] I think you don't need to import `useState`, Kindly remove it to keep the code simple.